.github/workflows/compare_translations.py

```diff
@@ -115,5 +119,18 @@ def check_translations():
 
 
 if __name__ == "__main__":
-    check_translations()
+    parser = argparse.ArgumentParser(
```

Comment:
1. Put all of this in a `main()` function for better readability
2. Remember the docstring

Reply:
I have made the changes suggested by you .